src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -57,6 +57,23 @@ public NodeAutoCompleteSearchControl()
             HomeWorkspaceModel.WorkspaceClosed += this.CloseAutoCompletion;
         }
 
+        public NodeAutoCompleteSearchControl(Window window, NodeAutoCompleteSearchViewModel viewModel)
```

Comment:
Can this be internal ?  I don't think we want it in the API.

Reply:
Done